### ISSUE TITLE
Visual hover state for links in body texts

### ISSUE THREAD

**Issue by xinemata** [Kcaden20/critical-code-cookbook#19]
For accessibility purposes, can we add a visual hover state for links inside the body texts?

I believe visual hover states have been implemented everywhere else on the site, so we should be all good on those fronts.  

<img width="539" alt="Screen Shot 2022-06-22 at 11 53 41 PM" src="https://user-images.githubusercontent.com/9159424/175225366-0d1ec697-e2a1-44b4-b2c4-24ecd0ad47e5.png">

**Comment by Kcaden20** [comment]
Visual Hover State has been added. Since the in-body text already has an underline, this visual state inverts the colors to highlight it—This also distinguishes it from nav related hover states and references the hover state of other linked items within the body. 

EDIT: Updated to include a little padding on the sides
![Screen Shot 2022-06-23 at 2 32 08 PM](https://user-images.githubusercontent.com/13590895/175370118-f4222f20-9000-4ba0-9fd4-65f633e613b4.png)

**Comment by xinemata** [comment]
Thanks. It looks good to me. Can you also apply it to links under `<ol>`, `<ul>`, and `<figcaption>`?

If you take a look at [this entry](https://criticalcode.recipes/contributions/critical-data-practice-at-home-and-with-friends), you can see that the visual effect doesn't apply to links under those tags.

**Comment by xinemata** [comment]
Found another one – `<h4>` under [How to code human bodies for devouring and mutual enlightenment](https://criticalcode.recipes/contributions/how-to-code-human-bodies-for-devouring-and-mutual-enlightenment ) doesn't have a visual hover state. 

<img width="732" alt="Screen Shot 2022-06-23 at 5 57 36 PM" src="https://user-images.githubusercontent.com/9159424/175434255-8738de07-1f1e-4b1f-85c6-a34346b8e260.png">

**Comment by xinemata** [comment]
Also links under subpages. See [this example](https://criticalcode.recipes/contributions/critical-data-practice-at-home-and-with-friends). 

<img width="1201" alt="Screen Shot 2022-06-23 at 6 35 50 PM" src="https://user-images.githubusercontent.com/9159424/175437273-1c460010-7086-4bdf-b69b-0e31b727971f.png">

**Comment by Kcaden20** [comment]
Alright, I was being too specific with my `css` selector. It's broader now to encompass any `a` tag within the `main` tag

**Comment by Kcaden20** [comment]
@xinemata This should be properly implemented now but will leave open in case something else comes up

**Comment by xinemata** [comment]
Didn't find any other missing spot during the second round of editing. Closing the issue.
